server/index.html

```diff
@@ -1151,11 +1151,11 @@
 							"O2": "mmail.co.uk",
 							"Republic Wireless": "text.republicwireless.com",
 							"Sprint": "messaging.sprintpcs.com",
-							"Straight Talk": "vtext.com",
 							"T-Mobile": "tmomail.net",
 							"Ting": "message.ting.com",
 							"U.S. Cellular": "email.uscc.net",
-							"Verizon Wireless": "vtext.com",
+							"Verizon Wireless / Straight Talk": "vtext.com",
```

Comment:
I believe Verizon Wireless and Straight Talk should be separate just because it's less work for a user to find Straight Talk alphabetically rather than to look for Verizon Wireless. They might not know that Verizon Wireless and Straight Talk have the same SMS email address. What do you think?

Reply:
This is a very valid point @Prince25.. On the one hand it does make sense to have the carriers listed separately since they are technically two different carriers. However it's also confusing when you go to the settings page and see a different carrier shown in the dropdown than what you selected previously. Perhaps something that checks both the carrier name and address when reading from the config to populate the dropdown, or there could be a note either in the docs or front-end that outlines some carriers share addresses and as such are grouped together in the dropdown. 

Thoughts?

Reply:
You, my friend, found a bug I wasn't aware of! The way it works is that it tries to match the SMS email address that's currently in `config.json` to the carrier. Since Straight Talk and Verizon Wireless have the same SMS email address, it matches with Straight Talk whether you chose Straight Talk or Verizon Wireless since Straight Talk comes first on the list. The same bug exists for AT&T and FirstNet. 

Your changes would solve this problem. I currently can't think of a simple way to solve this issue while keeping them separate. So for now, could you also group AT&T and FirstNet as `AT&T / FirstNet`?